README.md

```diff
@@ -188,6 +193,38 @@ func Connect() {
 }
 ```
 
+### Automatic IAM Database Authentication
+
+The Go Connector supports [Automatic IAM database authentication][].
+
+Make sure to [configure your AlloyDB Instance to allow IAM authentication][configure-iam-authn]
+and [add an IAM database user][add-iam-user].
+
+A `Dialer` can be configured to connect to an AlloyDB instance using
+automatic IAM database authentication with the `WithIAMAuthN` Option.
+
+```go
+d, err := cloudsqlconn.NewDialer(ctx, cloudsqlconn.WithIAMAuthN())
```

Comment:
This showcases Cloud SQL Go Connector...

Reply:
:facepalm: